eclass/zig-utils.eclass

```diff
@@ -459,8 +459,8 @@ zig-utils_find_installation() {
 		die "Could not find (suitable) Zig at \"${base_path}\""
 	fi
 
-	export ZIG_EXE="${selected_path}"
-	export ZIG_VER="${selected_ver}"
+	declare -g ZIG_EXE="${selected_path}"
```

Comment:
Do you actually need `declare -g`? I think just assigning the value will do the same.

Reply:
I like explicitness here, can confuse `ZIG_EXE=...` as assigning to local `ZIG_EXE`.